test/txn-test.js

```diff
@@ -0,0 +1,114 @@
+import { assert } from 'chai';
```

Comment:
Should this file be `txManager-test` to match with the specific things it's testing?

Reply:
unless you envision more tx related (non-manager) utilities being tested here too?

Reply:
Moved into separate files and updated the file names